test/evergreen.yml

```diff
@@ -296,12 +300,11 @@ functions:
           powershell.exe  -NonInteractive '.\test\evergreen\build_windows.ps1' -configure 1 $DEFINED_EVERGREEN_CONFIG_FLAGS ${windows_configure_flags|}
         else
           echo "Using config flags $DEFINED_EVERGREEN_CONFIG_FLAGS ${posix_configure_flags|}"
-          # Fetch the gperftools library if needed. This will also get tcmalloc.
-          if [[ ${ENABLE_TCMALLOC|0} -eq 1 ]] && [ ! -d "./automation-scripts" ]; then
-            is_cmake_build=true
-            git clone git@github.com:wiredtiger/automation-scripts.git
-            . automation-scripts/evergreen/find_gperftools.sh ${s3_access_key} ${s3_secret_key} ${build_variant} $is_cmake_build
+          
+          if [[ ${ENABLE_TCMALLOC|0} -eq 1 ]]; then
+            source test/evergreen/tcmalloc_install_or_build.sh ${build_variant}
```

Comment:
`source` is only useful when you intend to pull some definitions from the other file to the current one.
I think that in this case you should call it as an external program.

Reply:
I was following the convention of the pre-existing work simply replacing the less obvious period `.` with `source`.
While I appreciate what you're saying, I'm inclined to keep it as it does not require the execute bits to be set on the script file.